packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -11,6 +11,32 @@ const outputDir = Path.resolve(__dirname, '../dist/');
 
 const WIDGET_VERSION = findLatestWidgetVersion(signInWidgetMajorVersion);
 
+function configUris() {
+  let uris;
+  switch (process.env.ENV) {
+    case 'prod':
+      uris = {
+        baseUri: 'https://okta-devok12.okta.com',
+        registrationPolicyId: 'reg405abrRAkn0TRf5d6',
+        idps: {
+          github: '0oayfl0lW6xetjKjD5d5',
+          google: '0oay75bnynuF2YStd5d5',
+        }
+      }
+      return;
+    default:
```

Comment:
Consider adding the `case 'test':` here right above `default` and let it fall through for now so it's clear that we can expect `'test'` as a valid value for `ENV`

Reply:
@shawyu-okta  like this?
`` switch (process.env.DEPLOY_ENV) {
    case 'prod':
      uris = {
        baseUri: 'https://okta-devok12.okta.com',
        registrationPolicyId: 'reg405abrRAkn0TRf5d6',
        idps: {
          github: '0oayfl0lW6xetjKjD5d5',
          google: '0oay75bnynuF2YStd5d5',
        }
      }
      return;
    case 'test':
    default:
      uris = {
        baseUri: 'https://okta-dev-parent.trexcloud.com',
        registrationPolicyId: 'reg3kwstakmbOrIly0g7',
        idps: {
          github: '0oa3jobx2bBlylNft0g7',
          google: '0oa3jaktbqkiwCthn0g7',
        }
      }
  }``

Reply:
How you have it now is good